lookerpy/apis/dashboard_api.py

```diff
@@ -28,6 +28,7 @@
 
 from ..configuration import Configuration
 from ..api_client import ApiClient
+from .connect_to_api import connect_to_api as api_connection
```

Comment:
Why rename?

Reply:
(1) More preference than anything - 'connect_to_api' is a verb, 'api_connection' is a noun.  I'm using the return value of the function, so I thought it'd be a bit clearer to make it a noun.
(2) api_connection is actually a bit misleading, because right now the function just returns the access token.  I'll change this rename to account for that.

Reply:
Removed rename